lib/index.d.ts

```diff
@@ -738,37 +738,44 @@ declare namespace Stardog {
              * Stores a query in Stardog, either on the system level or for a given database.
              *
              * @param {Connection} conn the Stardog server connection
-             * @param {StoredQueryOptions} config an object specifying the options to set on the new query
-             * @param {object} params additional parameters if needed
+             * @param {StoredQueryOptions} storedQuery an object specifying the options to set on the new query
+             * @param {object} options additional options to customize query
```

Comment:
not sure this description is quite accurate-- it's more like additional options to customize the request, or just request options now

Reply:
[`d833110` (#291)](https://github.com/stardog-union/stardog.js/pull/291/commits/d833110c8d2444290c917019b3713e54eb4896f7) ?